zephyr/wrapper.c

```diff
@@ -678,15 +678,15 @@ void platform_dai_timestamp(struct comp_dev *dai,
 		posn->flags |= SOF_TIME_DAI_VALID;
 
 	/* get SSP wallclock - DAI sets this to stream start value */
-	posn->wallclock = k_cycle_get_64() - posn->wallclock;
+	posn->wallclock = sof_cycle_get_64() - posn->wallclock;
```

Comment:
I think this is the only place where we there is a stronger case for 64bit type. For host to compare timestamps, the added precision and very infrequent rollovers (making comparing timestamps easier across longer streams), adds value.